rollup/sync_service.go

```diff
@@ -221,6 +221,12 @@ func (s *SyncService) initializeLatestL1(ctcDeployHeight *big.Int) error {
 	} else {
 		log.Info("Found latest index", "index", *index)
 		block := s.bc.GetBlockByNumber(*index - 1)
+		if block == nil {
+			block = s.bc.CurrentBlock()
+			idx := block.Number().Uint64()
```

Comment:
Requesting to add a (probably useless) sanity check to ensure that `idx > index`.